app/components/profile/ProfileComponent.jsx

```diff
@@ -14,11 +14,12 @@ require('../signup/SignupComponent.css');
 class ProfileComponent extends React.Component {
     constructor(props) {
         super(props)
-        //const checkboxInterests = interests.map(interest => ({ interest, checked: false }));
         var checkInter = []
+
         if (this.props.location.state) {
             
             var volunteerInterests = this.props.location.state.userData.interests == 0 ? this.props.location.state.userData.volenteerInterests : this.props.location.state.userData.interests
```

Comment:
typo in volunteer

Reply:
actually that one needs to be there in case of crap legacy data which is why we will want to delete all data and re upload but will remove now.